File: LibraryApp/app/src/main/java/com/example/libraryapp/Locker.java
package com.example.libraryapp;

public class Locker {
    private String id;
    private String number;
    private boolean isRented;
    private String rentedBy;

    public Locker() {}

    public Locker(String id, String number, boolean isRented, String rentedBy) {
        this.id = id;
        this.number = number;
        this.isRented = isRented;
        this.rentedBy = rentedBy;
    }

    public String getId() { return id; }
    public String getNumber() { return number; }
    public boolean getIsRented() { return isRented; }
    public String getRentedBy() { return rentedBy; }
}

File: src/main/java/com/example/libraryapp20/Locker.kt
package com.example.libraryapp20

class Locker {
    var id: String? = null
        private set
    var number: String? = null
        private set
    var isRented: Boolean = false
        private set
    var rentedBy: String? = null
        private set

    constructor()

    constructor(id: String?, number: String?, isRented: Boolean, rentedBy: String?) {
        this.id = id
        this.number = number
        this.isRented = isRented
        this.rentedBy = rentedBy
    }
}